### PR TITLE
Fortest

### PR DESCRIPTION
**Pull request by Psisi**
please check.

### CODE REVIEW

File: flex1.html [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Flex</title>
+    <link rel="stylesheet" href="styleflex.css" />
+</head>
+<body>
+    <div class="container">
+        <div class="pricing-table">
+            
+            <div class="pricing-plan">
+                <img src="img/plane1.png" alt="paperplane" class="pricing-img">
```
Comment (jiangwalle) [inline flex1.html:15]: The images are missing from the PR. Please add them in the PR as well.

File: styleflex.css [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+.container {
+    background-color: white;
+    width: 1050px;
+    height: 680px;
+    margin: 3.5rem auto;
+    position: relative;
+}
+
+.pricing-table {
+    position: absolute;
```
Comment (jiangwalle) [inline styleflex.css]: What does "absolute" do here?

File: styleflex.css [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+    width: 1000px;
+    height: 650px;
+    margin: 1rem 1.7rem;
+    color:rgb(9, 118, 243);
```
Comment (jiangwalle) [inline styleflex.css]: Format: missing space between ":" and r

File: styleflex.css (hunk 2 above)
Comment (Psisi) [reply styleflex.css]: To make a margin

File: flex1.html (hunk 1 above)
Comment (Psisi) [reply flex1.html:15]: Screenshot (32)

Comment (Psisi) [reply flex1.html:15]: Screenshot (33)

File: styleflex.css [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+    height: 100%;
+    background-color: rgb(183, 224, 250);
+    font-size: 14px;
+    font-family: Verdana, Geneva, Tahoma, sans-serif;
+}
+
+.container {
+    background-color: white;
+    width: 1050px;
+    height: 680px;
```
Comment (jiangwalle) [inline styleflex.css]: It's not recommended to set height on the container. Instead, we can just let content inside the container to control its height. The benefit is once we want to update on the contents, we can just do it without worry about the container height.

File: styleflex.css [hunk 5]
```diff
@@ -0,0 +1,127 @@
...
+
+.pricing-plan:hover .pricing-button {
+    color: white;
+    background-color: rgb(26, 156, 236);
+}
+
+@media screen and (max-width: 700px) {
+    .container{
+        width: 100%;
+        height: 1950px;
```
Comment (jiangwalle) [inline styleflex.css]: Same as above. Set height on container level is not recommened.

**Pull request merged**